src/build/build.ts

```diff
@@ -88,6 +88,15 @@ export async function build(
     logger.info(`(${buildName}) Building...`);
   });
 
+  if (options.addPushManifest) {
+    buildStream = buildStream.pipe(polymerProject.addPushManifest());
+    if (!options.addServiceWorker) {
+      logger.warn('Push manifest generated without a service worker. ' +
+      'HTTP2/Push relies on service workers to prevent duplicate file ' +
```

Comment:
I looked into this more and you guys are right, there are a few different ways for the browser/server to negotiate to stop duplicate pushes. Removed in 2708c9b.